client/core/trade.go

```diff
@@ -182,14 +184,6 @@ func newTrackedTrade(dbOrder *db.MetaOrder, preImg order.Preimage, dc *dexConnec
 	return t
 }
 
-// rate returns the order's rate, or zero if a market or cancel order.
-func (t *trackedTrade) rate() uint64 {
```

Comment:
this is dead code, so, removing it

Reply:
My linters have caught that too.  Thanks.